src/test/java/com/epam/reportportal/cucumber/UtilsTest.java

```diff
@@ -0,0 +1,57 @@
+package com.epam.reportportal.cucumber;
+
+import com.epam.ta.reportportal.ws.model.ParameterResource;
+import cucumber.api.Argument;
+import org.junit.Test;
+import rp.com.google.common.collect.Lists;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+/**
+ * @author <a href="mailto:ihar_kahadouski@epam.com">Ihar Kahadouski</a>
+ */
+public class UtilsTest {
```

Comment:
Please also add an integration test to ensure you bypass parameter to the client.

Reply:
added